src/main/resources/i18n/messages.properties

```diff
@@ -556,3 +556,7 @@ reviewScore.four=4
 reviewScore.three=3
 reviewScore.two=2
 reviewScore.one=1
+chat.replay=replay
```

Comment:
This doesn't seem to be used anywhere and should be removed (otherwise, it should probably start uppercase)